### ISSUE TITLE
Improve Landing Page Accessibility

### ISSUE THREAD

**Issue by Zmwang622** [PittCSWiki/pittcswiki#331]
As, @GameDog9988 points out in #329 , our site does not work well on screen readers. 

> Just checked, not super good. Combines the alt, h3, and p into one long piece of barely intelligible gibberish. Labels the element as a "button group". Maybe we can at the very least add an aria-label to the button that points out that it links to the about page? Do a similar thing for the other buttons?
>
> If we don't want to have those alt tags for the images, we can replace the img elements with this: https://www.gatsbyjs.com/plugins/gatsby-plugin-svgr/
>
> And then add a title element to the SVGs like so: https://css-tricks.com/accessible-svgs/
> 
> Also in case you were curious, the screenreader I used (VoiceOver) did not reach the learn more links. It only recognizes the div buttons
